ONSdigital/eq-survey-runner
# Add descriptions to relationship options

### What is the context of this PR?
Adds descriptions to relationship options.

### How to review 
- Check the test_relationship schema works
- Check the census schemas have the correct content
